pensjon-brevbaker/src/main/kotlin/no/nav/pensjon/brev/maler/legacy/fraser/LegacyFunksjonsfraser.kt

```diff
@@ -0,0 +1,96 @@
+@file:Suppress("ClassName")
+
+package no.nav.pensjon.brev.maler.legacy.fraser
+
+import no.nav.pensjon.brev.api.model.maler.legacy.PE
+import no.nav.pensjon.brev.maler.legacy.*
+import no.nav.pensjon.brev.template.Expression
+import no.nav.pensjon.brev.template.LangBokmalNynorskEnglish
+import no.nav.pensjon.brev.template.Language.*
+import no.nav.pensjon.brev.template.ParagraphPhrase
+import no.nav.pensjon.brev.template.dsl.ParagraphOnlyScope
+import no.nav.pensjon.brev.template.dsl.expression.equalTo
+import no.nav.pensjon.brev.template.dsl.expression.greaterThan
+import no.nav.pensjon.brev.template.dsl.expression.lessThanOrEqual
+import no.nav.pensjon.brev.template.dsl.text
+
+// Fraser som var funksjoner i Exstream
+object LegacyFunksjonsfraser {
```

Comment:
Jeg har gjordt disse om til expression functions på min branch slik at du slipper å splitte opp text expressions for å bruke disse frasene. Opp til deg om du vil bruke disse.

Reply:
Det hadde vært nice! Skal jeg vente på at du blir ferdig eller hva tenker du?

Reply:
Skal si fra når jeg har merget mitt :)